src/main/java/com/three_tech_solutions/slot_app/services/implementations/PlanServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.three_tech_solutions.slot_app.services.implementations;
+
+import com.three_tech_solutions.slot_app.controllers.responses.PlanTypeResponse;
+import com.three_tech_solutions.slot_app.data.enums.PlanType;
+import com.three_tech_solutions.slot_app.data.mappers.PlanTypesMapper;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+
+
+@Service
+public class PlanServiceImpl implements com.three_tech_solutions.slot_app.services.interfaces.PlanService {
```

Comment:
Acá la interfaz te quedó mal

Debería quedar así
```suggestion
public class PlanServiceImpl implements PlanService {
```

y agregar el import al inicio del archivo

`import com.three_tech_solutions.slot_app.services.interfaces.PlanService`